### DISCUSSION TITLE
Cmd /scan not working

### DISCUSSION THREAD

**Discussion by sakulmore** [Zrips/CMI#6363] [Q&A]
Hello,

how do i use the command /scan ? It's somehow not working...

Video: https://youtu.be/x4akji0c2Bc

**Comment by mrfloris** [comment]
<img width="1110" alt="Screenshot 2021-11-15 at 14 09 03" src="https://user-images.githubusercontent.com/28841349/141787284-703c450e-1a3f-4fcc-a84d-09aed5d0b1eb.png">

Using this in 1.17.1 I can't say I am having much luck finding a chest I put in a chest in front of me. Using elvl, e, h, n, etc. They all return with no info. 

Might be a bug.

**Reply by mrfloris** [reply]
I see in the meantime you've reported it as such; https://github.com/Zrips/CMI/issues/6364

**Comment by mrfloris** [comment]
Now shows as fixed, haven't heard back - I assume this means this is resolved.
